src/order/compactly_generated.lean

```diff
@@ -231,4 +253,42 @@ begin
   exact λ x, ⟨{x}, ⟨λ x _, h x, Sup_singleton⟩⟩,
 end
 
+/-- A compact element k has the property that any b < k lies below a "maximal element below k",
+which is to say [⊥, k] is coatomic. -/
+theorem Iic_coatomic_of_compact_element (k : α) (hk : k ≠ ⊥) (h : is_compact_element k) :
```

Comment:
```suggestion
theorem Iic_coatomic_of_compact_element {k : α} (h : is_compact_element k) :
```
This should still be true for `k = ⊥`, as `set.Iic ⊥` is a one-element lattice, and those should be trivially atomic and coatomic, although I have not proven it.